packages/boxel-ui/addon/src/styles/variables.css

```diff
@@ -256,4 +257,15 @@
   --boxel-form-control-border-radius: var(--boxel-border-radius);
   --boxel-form-control-dark-mode-placeholder-color: #acacac;
   --boxel-form-control-dark-mode-border: var(--boxel-border-flexible);
+
+  /* deprecated */
+  --boxel-sp-xxxl: var(--boxel-sp-3xl); /* xxxl is deprecated, use 3xl */
+  --boxel-sp-xxl: var(--boxel-sp-2xl); /* xxl is deprecated, use 2xl */
+  --boxel-sp-xxs: var(--boxel-sp-2xs); /* xxs is deprecated, use 2xs */
+  --boxel-sp-xxxs: var(--boxel-sp-3xs); /* xxxs is deprecated, use 3xs */
+  --boxel-font-med: var(--boxel-font-md);
+  --boxel-font-size-med: var(--boxel-font-size-md);
+  /* med is deprecated, use md */
+  --boxel-font-size-xxl: var(--boxel-font-size-2xl);
+  /* xxl is deprecated, use 2xl */
```

Comment:
Maybe we should just get rid of them instead of having both versions?

Reply:
Maybe a separate PR since it turns out there are many of them